### PR TITLE
LJS-120 | Upgraded Froala Editor from SB-5 to SB-6

### PR DESCRIPTION
**Pull request by Priyanshu-Jaiswal**
In this PR, I have upgraded froala editor into sb-6

Thanks
Priyanshu Jaiswal

### CODE REVIEW

File: packages/froala-editor/src/components/index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    if (palleteType !== 'light') {
+      themeVal = palleteType;
+    }
+
+    return themeVal;
+  };
+
+  const [state, setState] = useState({ theme: getTheme(props), initControls: null });
+
+  /**
```
Comment (Rahulgarg30591) [inline packages/froala-editor/src/components/index.js:31]: Kindly remove the commented code.

**Review by Rahulgarg30591 — CHANGES_REQUESTED** [review]

File: packages/froala-editor/src/components/index.js (hunk 1 above)
Comment (Priyanshu-Jaiswal) [reply packages/froala-editor/src/components/index.js:31]: Ok

File: packages/froala-editor/package.json [hunk 2]
```diff
@@ -40,22 +40,23 @@
...
     "react-dom": "^16.13.1",
     "react-test-renderer": "^16.13.1",
     "rimraf": "^2.7.0",
-    "storybook-readme": "^4.0.5",
+    "storybook-readme": "^5.0.8",
     "typeface-montserrat": "0.0.75",
     "typeface-roboto": "0.0.75"
   },
   "dependencies": {
+    "@storybook/addon-essentials": "^6.0.28",
```
Comment (Rahulgarg30591) [inline packages/froala-editor/package.json]: All StoryBook dependencies should come inside dev dependencies.

**Review by Rahulgarg30591 — CHANGES_REQUESTED** [review]

File: packages/froala-editor/package.json (hunk 2 above)
Comment (Priyanshu-Jaiswal) [reply packages/froala-editor/package.json]: All things fixed

**Pull request merged**